Sources/EpoxyCollectionView/Models/ItemModel/AnyItemModel.swift

```diff
@@ -68,6 +72,9 @@ extension AnyItemModel: InternalItemModeling {
 
   public func configure(cell: ItemWrapperView, with metadata: ItemCellMetadata) {
     model.configure(cell: cell, with: metadata)
+    if let view = cell.view {
+      setContent?(.init(view: view, metadata: metadata))
+    }
```

Comment:
It would be nice to support `setBehavior` and `configureStateChange` for symmetry using the same approach, up to you if you'd like to include this in this change or not

Reply:
Meaning that `configureStateChange` and `setBehavior` would perform similar setContent updates?

Reply:
@julianozen specifically that we'd support `SetBehaviorsProviding` and `DidChangeStateProviding` in the same way on `AnyItemModel`. Up to you if you'd like to include this in this PR 👍